Scripts/CI/README_Metadata_StyleCheck/entry.py

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/env python3
+
+import os
+import json
+import argparse
+import subprocess as sp
+
+# A set of category folder names in current sample viewer.
+# Only run the checks when a file path is within one of these category folders.
+categories = {
+    'analysis',
+    'display_information',
+    'editing',
+    'feature_layers',
+    'geometry',
+    'group_layers',
+    'hydrography',
+    'image_layers',
+    'kml',
+    'local_server',
+    'map',
+    'map_view',
+    'network_analysis',
+    'ogc',
+    'portal',
+    'raster',
+    'scene',
+    'search',
+    'symbology',
+    'tiled_layers',
+    'utility_network'
+}
```

Comment:
I remember we discussed a solution for this. At the moment this list is repeated in `README_style_checker.py`.
Perhaps we can store it in a text file `categories.txt` and read it in in both scripts, that way there's only one source of truth?

Reply:
Definitely, I've added this along with a few of the other improvements that we discussed to an issue in CS repo for future work to improve. I'll do the tidy ups from the code reviews and leave anything that will need more thought/time to that issue for another iteration.